### PR TITLE
Cleanup getting started page

### PR DESCRIPTION
**Pull request by henri-tremblay**
* Remove user managed cache from getting-started. 
* Present a full xml including the namespaces. 
* Talk about XML right after Java. 
* Other minor formatting on getting-started

### CODE REVIEW

File: docs/src/docs/asciidoc/user/getting-started.adoc [hunk 1]
```diff
@@ -60,6 +58,40 @@ It only takes one parameter which is the key, and returns the value associated w
 <10> In order to release all transient resources (memory, threads, ...) a `CacheManager` provides to `Cache` instances
      it manages, you have to invoke `CacheManager.close()`, which in turns closes all `Cache` instances known at the time.
 
+[[configuring-with-xml]]
+=== Configuring With XML
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: > XML configuration

to match the previous section?

File: docs/src/docs/asciidoc/user/migration-guide.adoc [hunk 2]
```diff
@@ -18,8 +18,7 @@ This guide provides sample code snippets to help migrate Ehcache 2.x code to Ehc
 
 Per mapping expiry covers use cases where a subset of mappings have different expiration settings than the ones configured at the cache level.
 
-
-==== Ehcache 2.x Code snippet:
+=== Ehcache 2.x Code snippet
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/migration-guide.adoc]: Make sure the removal of the `:` does not cause a change in the section id. If it does, explicitly set it to the older value.

File: docs/src/docs/asciidoc/user/getting-started.adoc [hunk 3]
```diff
@@ -80,23 +112,6 @@ include::{sourcedir33}/clustered/client/src/test/java/org/ehcache/clustered/clie
 
 NOTE: See <<clustered-cache.adoc#,the clustered cache documentation>> for more information on this feature.
 
-==== User managed cache
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: Can we have a section id down in the document that matches that old one and links to the user managed doc page? That would make sure we do not break any existing link to it

File: docs/src/docs/asciidoc/user/migration-guide.adoc [hunk 4]
```diff
@@ -32,7 +31,7 @@ include::{sourcedir33}/impl/src/test/java/org/ehcache/docs/Ehcache2.java[tag=Cus
 <3> Only setting the computed expiry on element, if other than default expiry.
 <4> Waiting for 2.1 seconds - assuming 2 seconds is the custom expiry duration - to get the mapping to be expired.
 
-==== Corresponding Ehcache 3.x Code Snippet:
+=== Corresponding Ehcache 3.x Code Snippet
```
Comment (ljacomet) [inline docs/src/docs/asciidoc/user/migration-guide.adoc]: Same comment as the other one in this file

File: docs/src/docs/asciidoc/user/getting-started.adoc (hunk 1 above)
Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/getting-started.adoc]: Absolutly

File: docs/src/docs/asciidoc/user/migration-guide.adoc (hunk 2 above)
Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/migration-guide.adoc]: Just checked. The colon wasn't in the archor before

File: docs/src/docs/asciidoc/user/migration-guide.adoc (hunk 4 above)
Comment (henri-tremblay) [inline docs/src/docs/asciidoc/user/migration-guide.adoc]: See above

**Comment by henri-tremblay** [comment]
No daggy fix was dedided. Now rebased.

**Pull request merged**